projects/dotcms-field-elements/src/components/dot-key-value/key-value-table/key-value-table.e2e.ts

```diff
@@ -0,0 +1,35 @@
+import { newE2EPage, E2EElement, E2EPage } from '@stencil/core/testing';
+import { EventSpy } from '@stencil/core/dist/declarations';
+
+describe('key-value-table', () => {
+    let page: E2EPage;
+    let element: E2EElement;
+    let spyDeleteItemEvent: EventSpy;
+
+    beforeEach(async () => {
+        page = await newE2EPage();
+        await page.setContent(`<key-value-table />`);
+        element = await page.find('key-value-table');
+        element.setProperty('values', [{ keyA: '1', keyB: '2' }]);
+        await page.waitForChanges();
+    });
+
+    it('renders', async () => {
+        // tslint:disable-next-line:max-line-length
+        const expectedMarkup = `<key-value-table class=\"hydrated\"><table><tbody><tr><td><button type=\"button\" id=\"undefined_undefined_0\" class=\"dot-key-value__delete__button\"><label for=\"undefined_undefined_0\">Delete</label></button></td><td></td><td></td></tr></tbody></table></key-value-table>`;
+        expect(element.outerHTML).toBe(expectedMarkup);
+    });
+
+    describe('Events', () => {
+        beforeEach(async () => {
+            spyDeleteItemEvent = await page.spyOnEvent('deleteItemEvt');
+        });
+
+        it('should emit "deleteItemEvt" when deleting an item', async () => {
```

Comment:
Are we testing that is rendering the rows correctly when add and remove?

Reply:
https://github.com/dotCMS/core-web/blob/96894a92b2299c428f66d30fdf342146475e256a/projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.e2e.ts#L76
https://github.com/dotCMS/core-web/blob/96894a92b2299c428f66d30fdf342146475e256a/projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.e2e.ts#L119